dynawo/3rdParty/libiidm/libiidm.cmake

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+cmake_minimum_required(VERSION 3.12)
+
+include($ENV{DYNAWO_HOME}/dynawo/cmake/CPUCount.cmake)
+if(NOT DEFINED CPU_COUNT)
+  message(FATAL_ERROR "CPUCount.cmake: file not found.")
+endif()
+
+set(packet_name        "libiidm")
```

Comment:
packet->package

Reply:
Please do not close a discussion if it will not be solved in the next push

Reply:
Sorry, I made an error translating. Preceding message asked to put an english word. I did, so I closed conversation. That is a new error I did not even notice. I change the term.